locator/token_metadata.cc

```diff
@@ -937,6 +1004,19 @@ std::multimap<inet_address, token> token_metadata_impl::get_endpoint_to_token_ma
     return cloned;
 }
 
+topology_change_info::topology_change_info(token_metadata target_token_metadata, std::vector<dht::token> all_tokens,
+        bool is_read_new_requested)
```

Comment:
Lets not use the same names for parameters and for class members. If one will want to access a field in the body later it will do not what one would expect.

Reply:
is there any naming convention for these cases?

Reply:
Underscore at the end of a parameter is the most common one I think. `is_read_new_requested_`.

Reply:
fixed